clients/tabby-agent/src/TabbyAgent.ts

```diff
@@ -140,6 +141,12 @@ export class TabbyAgent extends EventEmitter implements Agent {
     if (options.config) {
       await this.updateConfig(options.config);
     }
+    if (!this.config.anonymousUsageTracking.disable) {
+      await anonymousUsageLogger.event({
```

Comment:
is this the only event we're logging for now?

Reply:
Yes